llumnix/backends/vllm/scheduler.py

```diff
@@ -205,6 +206,25 @@ def schedule(self) -> Tuple[List[SequenceGroupMetadata], SchedulerOutputs]:
         self.update_instance_info_callback(self._get_instance_info())
         return seq_group_metadata_list, scheduler_outputs
 
+    def _schedule_running(self, *args, **kwargs):
+        args_list = list(args)
+        args_list[0] = self.running.copy()
+        remove_running = []
```

Comment:
Is `remove_running`a list to store not scheduled running requests? We need to change running queue every step, it may cause performance issue when running queue is large, each `args_list[0].remove()` is O(n), and use loop to remove is  O(n^2). We can use a `new_running_queue` to store running queue to be scheduled and return the original running queue.